app/controllers/connections_controller.rb

```diff
@@ -1,6 +1,10 @@
 # frozen_string_literal: true
 
 class ConnectionsController < ApplicationController
+  def index
+    @connections = sorted Connection.includes(:port, cable: :connections).limit(20)
```

Comment:
```suggestion
    @connections = sorted Connection.includes(:port, cable: :connections)
```

Add pagination like done for `ChangelogEntry`